js/src/admin/index.js

```diff
@@ -1,6 +1,6 @@
-import app from 'flarum/app';
+import app from 'flarum/common/app';
```

Comment:
This isn't shared between forum and admin, so we should use `admin` here.

```suggestion
import app from 'flarum/admin/app';
```